crates/hotshot/src/traits/networking.rs

```diff
@@ -163,14 +161,9 @@ impl Label for NetworkingMetrics {
 impl NetworkingMetricsValue {
     /// Create a new instance of this [`NetworkingMetricsValue`] struct, setting all the counters and gauges
     #[must_use]
-    pub fn new() -> Self {
-        let values = Arc::default();
-        let metrics: Box<dyn Metrics> = Box::new(NetworkingMetrics {
-            prefix: String::new(),
-            values: Arc::clone(&values),
-        });
+    #[allow(clippy::borrowed_box)]
+    pub fn new(metrics: &Box<dyn Metrics>) -> Self {
```

Comment:
Nit: you can do `&dyn Metrics` here to fix the clippy lint, and if the caller has `metrics: Box<dyn Metrics>`, they will be able to pass it as `&*metrics`. Just a style thing, not really required to fix this

Reply:
That's cleaner.